scripts/live_test/upload.py

```diff
@@ -0,0 +1,38 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import sys
+import os
+import datetime
+
+ARTIFACT_DIR = sys.argv[1]
+ACCOUNT_KEY = sys.argv[2]
+USER_LIVE = sys.argv[3]
+
+
+def main():
+    # Create container
+    date = datetime.datetime.now().strftime('%Y%m%d%H%M%S')
+    if USER_LIVE == '--live':
+        mode = 'live'
+    elif USER_LIVE == '':
+        mode = 'replay'
+    else:
+        mode = ''
+    container = date + mode
+    cmd = 'az storage container create -n {} --account-name clitestresultstac --account-key {} --public-access container'
```

Comment:
How often will this script get run? And it creates a container every time?

Reply:
Each time the pipeline is triggered. Yes, it does.